src/ServiceControl.Config/UI/InstanceAdd/MonitoringAddAttachment.cs

```diff
@@ -68,6 +68,19 @@ async Task Add()
                 ServiceAccountPwd = viewModel.Password
             };
 
+            if ((instanceMetadata.TransportPackage.Name == TransportNames.RabbitMQClassicConventionalRoutingTopology ||
+                 instanceMetadata.TransportPackage.Name == TransportNames.RabbitMQQuorumConventionalRoutingTopology ||
+                 instanceMetadata.TransportPackage.Name == TransportNames.RabbitMQClassicDirectRoutingTopology ||
+                 instanceMetadata.TransportPackage.Name == TransportNames.RabbitMQQuorumDirectRoutingTopology) &&
```

Comment:
You can probably encapsulate this as an extension method on the `TransportInfo` class and reuse it.